pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>dev.sancraft</groupId>
     <artifactId>stacker</artifactId>
-    <version>1.1-SNAPSHOT</version>
```

Comment:
not 1.1.1?

Reply:
KP

Reply:
Wir lassen es glaube erstmal nur auf Main